web/src/pages/account-setup.tsx

```diff
@@ -69,7 +69,7 @@ const AccountSetupPage = (): ReactElement => {
 
       await updateQueue.queue(userDataWithUser).update();
       analytics.event.finish_setup_on_myNewJersey_button.submit.go_to_myNJ_registration();
-      onSelfRegister({ router, updateQueue, userData: userDataWithUser, setRegistrationStatus });
+      router && onSelfRegister({ router, updateQueue, userData: userDataWithUser, setRegistrationStatus });
```

Comment:
Same `analytics.event` concern, but maybe group this check in the `if` at the top of the block? e.g.
```js
if (!updateQueue || !router) return;
```